tasks/install.yml

```diff
@@ -0,0 +1,80 @@
+---
+
+- name: Download latest wordpress version
+  get_url:
+    url: https://wordpress.org/latest.zip
+    dest: /tmp/wordpress_latest.zip
+
+- name: Unarchive wordpress zip to /var/www
+  unarchive:
+    src: /tmp/wordpress_latest.zip
+    dest: /var/www/
+    owner: www-data
+    group: www-data
+
+- name: Copy theme
```

Comment:
Describe this theme copying mechanism in README.

Reply:
done